### PR TITLE
Tcp nodelay

### PR DESCRIPTION
**Pull request by trobert**

### CODE REVIEW

**Review by gaspb — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+Global / version := "0.0.22-cdk"
```
Comment (guizmaii) [inline build.sbt:1]: The version should be taken from the Github release

**Review by sderosiaux — APPROVED** [review]
just to know, is there a PR ongoing on the original repo?

File: modules/core/src/main/scala/net/BitVectorSocket.scala [hunk 2]
```diff
@@ -81,7 +81,7 @@ object BitVectorSocket {
     sslOptions:   Option[SSLNegotiation.Options[F]],
   ): Resource[F, BitVectorSocket[F]] =
     for {
-      sock  <- sg.client[F](new InetSocketAddress(host, port))
+      sock  <- sg.client[F](new InetSocketAddress(host, port), noDelay = true)
```
Comment (guizmaii) [inline modules/core/src/main/scala/net/BitVectorSocket.scala:84]: Why not just submitting a PR to Rob Norris?

Comment (trobert) [reply modules/core/src/main/scala/net/BitVectorSocket.scala:84]: I can but not sure he will republish a 0.0.x (CE2 branch)

**Review by mbaechler — APPROVED** [review]
cool

**Pull request merged**